test/services/workarea/build_kit_variants/params_test.rb

```diff
@@ -154,6 +154,33 @@ def test_components
           component[:details]
         )
       end
+
+      def test_sku_based_components
+        params = {
+          '1' => {
+            product_id: 'PROD1',
+            quantity: 1,
+            skus: %w(SKU1-1)
+          }
+        }
+
+        components =
+          Params.new(bundled_products: products, components: params).components
```

Comment:
In my head I call this "the Duffy newline"

Reply:
ha. is this weird? Feel like its more readable and less lines than

```
components = Params.new(
  bundled_products: products, 
  components: params
).components
```

The call at the end make that format awkward. It also artificially pumps up your LOC which we all know is an important metric